Makefile

```diff
@@ -481,7 +481,7 @@ endif
 	rm -fr $(BUILDROOT)/julia-$(JULIA_COMMIT)
 
 # this target does not accept BUILDROOT
-light-source-dist.tmp: $(JULIAHOME)/doc/_build/html
+light-source-dist.tmp: $(BUILDROOT)/doc/_build/html/en/index.html
```

Comment:
Any idea why there's a comment above this that says this target doesn't accept `BUILDROOT`?  If that's no longer true, we should get rid of it.  I believe @vtjnash was the original author of that comment.